config/environments/review.rb

```diff
@@ -1,7 +1,7 @@
 require Rails.root.join("config/environments/production")
 
 Rails.application.configure do
-  config.log_level = :debug
+  config.log_level = :info
```

Comment:
Just noticed we have 2 lines for `config.log_level` on `config/environments/production.rb`

Reply:
Good catch @leandroalemao !
Very strange.
I have changed it to what we have for ECF app.